<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arm</class>
 <widget class="QMainWindow" name="Arm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ArmConfig">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ArmConfig</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>length1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>length2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>height2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>height1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="length1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="height3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="length2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="height4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Target">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Target</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetY">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>AngelOfLinks</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="angel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="angel2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="angel4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="angel3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Config">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sim">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Sim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawCircle">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawCircle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="R">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetZ">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>StartPoint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EndPoint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startx">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="starty">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="endx">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="endy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Move">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/arm/arm.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>16</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/ico/target_s.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/ico/run_s.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/ico/config_s.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Auto">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ifAuto">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>auto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu4_Free_Arm">
    <property name="title">
     <string>4-Free-Arm</string>
    </property>
   </widget>
   <addaction name="menu4_Free_Arm"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
